jf_agent/main.py

```diff
@@ -178,7 +181,7 @@ def main():
                 )
                 if issues_to_scan:
                     print_missing_repos_found_by_jira(config, creds, issues_to_scan)
-                return
+                return something_went_wrong
```

Comment:
`something_went_wrong` is always `False` at this point, right?  thus far it's only ever set to `True` if its in validate mode, in which case we don't get here.  I think?